### PR TITLE
fix(scripts): modify labels for obproxy set up from the shell

### PR DESCRIPTION
**Pull request by powerfooI**

### CODE REVIEW

File: scripts/setup-obproxy.sh [hunk 1]
```diff
@@ -225,6 +263,9 @@ spec:
               name: "sql"
             - containerPort: 2884
               name: "prometheus"
+          envFrom:
+            - configMapRef:
+                name: $CONFIG_MAP
```
Comment (chris-sun-star) [inline scripts/setup-obproxy.sh:268]: what if CONFIG_MAP is empty

Comment (powerfooI) [reply scripts/setup-obproxy.sh:268]: The CONFIG_MAP can not be empty. If `--cm` is given and is empty the script will throw an error. If `--cm` and `--env` are all given, it throws an error. If only `--env` is given, the script create a new config map.

Comment (chris-sun-star) [reply scripts/setup-obproxy.sh:268]: but what if none of --env or --cm is passed, I suspect the config map is only created when either of these parameters is passed

**Review by chris-sun-star — APPROVED** [review]
LGTM

**Pull request merged**